src/EventListener/DeprecationListener.php

```diff
@@ -14,49 +15,22 @@
 use Symfony\Component\HttpKernel\Event\ControllerEvent;
 
 /**
- * Class ControllerListener.
+ * Class DeprecationListener.
  *
  * Remember, what controller got called in this request, so we can get the corresponding annotation in the ResponseView.
- *
- * @package Shopping\ApiTKDeprecationBundle\EventListener
  */
 class DeprecationListener
 {
-    /**
-     * @var bool
-     */
-    private $masterRequest = true;
-
-    /**
-     * @var Reader
-     */
-    private $reader;
+    private bool $masterRequest = true;
```

Comment:
According to symfony/http-foundation we should imho use `$mainRequest` for new Code if this does not make problems